script.js

```diff
@@ -198,14 +182,8 @@ const getStudentList = () => {
 
 // Funzione per ottenere uno specifico studente
 const getStudent = (id) => {
-  for (let i = 0; i < students.length; i++) {
-    if (students[i].id == id) {
-      console.log(`student with id ${id} found in students.`);
-      return students[i]
-    };
-  }
-  console.log(`id ${id} not found in students`);
-  return null;
+  const students = getStudentList();
```

Comment:
In realtà dovremmo semplicemente disfarci della funzione `getStudentList()`, mi ricordo il discorso di usare correttamente la funzione che ritorna la lista se dobbiamo prendere un elemento specifico ma visto che secondo me andrebbe semplicemente rimossa allora questo lo farei come prima e poi eventualmente le funzioni `getstudentList` e `getRegisterList` le eliminiamo e basta.

Reply:
Ok visto che adesso `getStudentList()` prende i dati dal localStorage per me va bene